enaml/src/dynamicscope.cpp

```diff
@@ -663,6 +663,29 @@ DynamicScope_getitem( DynamicScope* self, PyObject* key )
     return 0;
 }
 
+PyObject*
+DynamicScope_get( DynamicScope* self, PyObject* args)
+{
+    PyObject *key;
+    PyObject *default_value = NULL;
+
+    if (!PyArg_ParseTuple(args, "O|O", &key, &default_value))
+        return cppy::type_error( args, "str" );
```

Comment:
Please add enclosing { }. I try to avoid single line if since they can lead to subtle bugs.